citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -141,11 +145,36 @@ $(function() {
 		var authId = $(this).attr('data-authority-id');
 		$("#uriEditorLocalId").val(authId);
 		$("#editorAuthorityUsed").html("Using stored authority entry <i>" + $(this).attr('data-authority-name') + "</i>.");
+		showPersonNameInModal($(this).attr('data-authority-name'), "Editor");
 		$("#uriLoadingFoundEditor").popover('hide');
 		event.preventDefault();
 	});
 });
 
+function showPersonNameInModal(name, personType){
+	var parsedName = name;
+	if(name.includes("(") && name.includes(")")) {
```

Comment:
what case is covered here? this needs some comments

Reply:
put an example in your comment. why do you cut out only what's in the bracket and not what follows the brackets? can the name continue after the brackets?